tensorboard/context.py

```diff
@@ -46,11 +51,23 @@ def __init__(self, auth=None):
           and will be replaced with default values if appropriate.
         """
         self._auth = auth if auth is not None else auth_lib.AuthContext.empty()
+        self._remote_ip = remote_ip
+        self._x_forwarded_for = (
+            x_forwarded_for if x_forwarded_for is not None else []
+        )
```

Comment:
Optional, but this can just be `x_forwarded_for or []` if you want
(or `x_forwarded_for or ()` for tuple).

Reply:
done.